docs/reference/commandline/run.md

```diff
@@ -134,89 +134,82 @@ previous changes intact using `docker start`. See `docker ps -a` to view a list
+The `docker run` command is powerful and has many options which can augment or
+override your container's Dockerfile settings at runtime. See the
+[Docker run reference](../run.md) for more thorough examples using the
+`docker run` command. The following are only a few examples.
 
-### Capture container ID (--cidfile)
+### Simple examples
 
-    $ docker run --cidfile /tmp/docker_test.cid ubuntu echo "test"
+Run the `hello-world` container to test your Docker installation.
 
-This will create a container and print `test` to the console. The `cidfile`
-flag makes Docker attempt to create a new file and write the container ID to it.
-If the file exists already, Docker will return an error. Docker will close this
-file when `docker run` exits.
+```bash
+$ docker run hello-world
+```
 
-### Full container capabilities (--privileged)
+Run the `sh` command in an `alpine` container in interactive mode. Remove the
+container when it stops. To stop the container, type `exit`.
 
-    $ docker run -t -i --rm ubuntu bash
-    root@bc338942ef20:/# mount -t tmpfs none /mnt
-    mount: permission denied
+```bash
+docker run --rm -it alpine sh
+```
 
-This will *not* work, because by default, most potentially dangerous kernel
-capabilities are dropped; including `cap_sys_admin` (which is required to mount
-filesystems). However, the `--privileged` flag will allow it to run:
+Run a `nginx` container, mapping port 80 to port 8080 on the host machine. To
+access the web server, go to http://localhost:8080/.
 
-    $ docker run -t -i --privileged ubuntu bash
-    root@50e3f57e16e6:/# mount -t tmpfs none /mnt
-    root@50e3f57e16e6:/# df -h
-    Filesystem      Size  Used Avail Use% Mounted on
-    none            1.9G     0  1.9G   0% /mnt
+```bash
+$ docker run -d -p 8080:80 nginx
+```
 
-The `--privileged` flag gives *all* capabilities to the container, and it also
-lifts all the limitations enforced by the `device` cgroup controller. In other
-words, the container can then do almost everything that the host can do. This
-flag exists to allow special use-cases, like running Docker within Docker.
+### More complex examples
 
-### Set working directory (-w)
 
-    $ docker  run -w /path/to/dir/ -i -t  ubuntu pwd
+#### Full container capabilities (--privileged)
 
-The `-w` lets the command being executed inside directory given, here
-`/path/to/dir/`. If the path does not exist it is created inside the container.
+```bash
+$ docker run --rm --it ubuntu bash
+root@bc338942ef20:/# mount -t tmpfs none /mnt
+mount: permission denied
+```
 
-### Set storage driver options per container
+Trying to run the `mount` command from within an unprivileged container results in
+an error, because by default, most potentially dangerous kernel capabilities are
+dropped, including `cap_sys_admin` (which is required to mount filesystems).
+Running the same command inside a privileged container started with the
+`--privileged` flag allows the `mount` command to succeed.
```

Comment:
I don't think we should use `--privileged` in any examples, can we just use `--cap-add sys_admin` instead?